src/DaPigGuy/PiggyCustomEnchants/enchants/armor/CloakingEnchant.php

```diff
@@ -24,14 +24,6 @@ class CloakingEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Cloaking";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 22, so I commented this line